resources/assets/less/beatmaps.less

```diff
@@ -264,6 +263,7 @@
 			position: absolute;
 			top: 45px;
 			left: 10px;
+			z-index: @z-index--beatmaps-panel - 2;
```

Comment:
It's still centralised! Just those layers are relative to that central z-index >_>

(design's hacky atm, will redo it later anyway)